<launch>
	<!--node pkg="imu_ultrasonic_data" type="imu_data_node" name="" output="screen"/-->

	<!--node pkg="imu_ultrasonic_data" type="ultrasonic_data_node" name="" -->
		<!--param name="ultrasonic_port" type="string" value="/dev/ttyUSB0"/-->
	<!--/node-->

	<node pkg="line_detect" type="front_left_global_node" name="" output="screen">
		<param name="pitch0" type="double" value="0.0"/>
		<param name="roll0" type="double" value="0.0"/>
		<param name="yaw0" type="double" value="0.0"/>
	</node>
	
</launch>


